src/molecules/Footers/OppskriftFooter.js

```diff
@@ -5,46 +5,31 @@ import { Col, Row, Grid } from '../..';
 import { SvgIcon } from '../../atoms/SvgIcon';
 
 const FooterWrapper = styled.footer`
-	background-color: ${props => props.theme.colors.darkGrey};
+	background-color: ${props => props.theme.colors.type};
 	color: ${props => props.theme.colors.white};
-	padding: 10.3rem 0 6.2rem;
 	font-family: ${props => props.theme.variables.mainFont};
-	font-size: 1.4rem;
-	letter-spacing: 2px;
-	
-	@media screen and (max-width: ${props => props.theme.flexboxgrid.breakpoints.md}rem) {
-		font-size: 3.2rem;
-		background-color: ${props => props.theme.colors.type};
-		padding-top: 14rem;
-		
-		.margin-bottom-sm-3{
-			margin-bottom: 3rem;
-		}
-		.margin-bottom-sm-5{
-			margin-bottom: 3rem;
-		}
-		.margin-bottom-sm-6{
-			margin-bottom: 3rem;
-		}
-		.margin-bottom-sm-10{
-			margin-bottom: 3rem;
-		}
+	font-size: ${props => props.theme.variables.uiRegularSize};
+	line-height: ${props => props.theme.variables.headingMediumLineHeight};
+	padding: 5rem 2rem 4rem;
+
+	@media screen and (min-width: ${props => props.theme.flexboxgrid.breakpoints.md}em) {
+		padding: 10rem 0 6rem;
 	}
 	
 	.text-left{
 		text-align: left;
 	}
 `;
 
+const LeftAlignedCol = Col.extend`
+	text-align: left;
```

Comment:
Isn't this the initial value?

Reply:
nope, it was centered by default for some reason
but that's a good point, i'm going to investigate it

Reply:
for some reason, when specifying `<Row center="md">`, text inside `Col` is also centered. Not sure if this is desired behavior, but anyway